app/actions/definitions/navigation.tsx

```diff
@@ -57,6 +57,7 @@ export const navigateToRecentSearchQuery = (searchQuery: SearchQuery) =>
     perform: () => history.push(searchPath(searchQuery.query)),
   });
 
+// could be a document action
```

Comment:
We need to assign a section to search.

<img width="951" alt="Screenshot 2022-03-11 at 2 07 23 AM" src="https://user-images.githubusercontent.com/40148448/157750264-66586537-3126-4acb-8b90-7e82472f0c25.png">